### PR TITLE
Bug 2049638: Add tooltip explanation on 'Select migration network' button

### PR DESCRIPTION
**Pull request by mturley**
On the VMware Hosts table, when no other tooltip needs to be displayed on the "Select migration network" button (other tooltips are "Select at least one host" or "Selected hosts have no networks in common"), the following tooltip will be shown:

Screen Shot 2022-02-28 at 3 32 39 PM

### CODE REVIEW

**Comment by mturley** [comment]
cc @stoobie

**Review by gildub — APPROVED** [review]
LGTM

File: pkg/web/src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx [hunk 1]
```diff
@@ -136,12 +135,13 @@ export const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHo
...
         <LevelItem>
-          <ConditionalTooltip
-            isTooltipEnabled={commonNetworkAdapters.length === 0}
+          <Tooltip
             content={
               selectedItems.length === 0
                 ? 'Select at least one host'
-                : 'Selected hosts have no networks in common'
+                : commonNetworkAdapters.length === 0
+                ? 'Selected hosts have no networks in common'
```
Comment (stoobie) [inline pkg/web/src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx:143]:
```suggestion
                ? 'Selected hosts have no network in common'
```

File: pkg/web/src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx [hunk 2]
```diff
@@ -136,12 +135,13 @@ export const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHo
...
+                : 'Select a network for migration data transfer for the selected hosts'
```
Comment (stoobie) [inline pkg/web/src/app/Providers/components/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx:144]:
```suggestion
                : 'Select a network for transferring migration data from the selected hosts'
```

**Review by stoobie — CHANGES_REQUESTED** [review]
Two suggestions.

**Pull request merged**